feeadjuster/feeadjuster.py

```diff
@@ -189,9 +224,10 @@ def feeadjust(plugin: Plugin):
     Otherwise, the plugin keeps the fees up-to-date.
     """
     plugin.mutex.acquire(blocking=True)
-    peers = plugin.rpc.listpeers()["peers"]
+    plugin.peers = plugin.rpc.listpeers()["peers"]
+    plugin.channels = plugin.rpc.listchannels()['channels']
```

Comment:
This will contain all the channels in the Lightning network? The `listchannels` currently returns 85039 channels for me. How much memory will this consume?

Reply:
yep, I can change it to only do this if `median` is used